mozilla-release/browser/components/nsBrowserGlue.js

```diff
@@ -1222,47 +1192,17 @@ BrowserGlue.prototype = {
 
     // Perform default browser checking.
     if (ShellService) {
-<<<<<<< HEAD
-#ifdef DEBUG
-      let shouldCheck = false;
-#else
-      let shouldCheck = ShellService.shouldCheckDefaultBrowser;
-#endif
-      let promptCount = 0;
-#ifdef RELEASE_BUILD
-#if 0
-// In CLIQZ we never want the prompt counter to take any effect.
-      promptCount =
-        Services.prefs.getIntPref("browser.shell.defaultBrowserCheckCount");
-#endif
-      let skipDefaultBrowserCheck =
-        Services.prefs.getBoolPref("browser.shell.skipDefaultBrowserCheck");
-#else
-||||||| merged common ancestors
-#ifdef DEBUG
-      let shouldCheck = false;
-#else
-      let shouldCheck = ShellService.shouldCheckDefaultBrowser;
-#endif
-      let promptCount = 0;
-#ifndef RELEASE_BUILD
-      promptCount =
-        Services.prefs.getIntPref("browser.shell.defaultBrowserCheckCount");
-      let skipDefaultBrowserCheck =
-        Services.prefs.getBoolPref("browser.shell.skipDefaultBrowserCheck");
-#else
-=======
       let shouldCheck = AppConstants.DEBUG ? false :
                                              ShellService.shouldCheckDefaultBrowser;
       let promptCount;
->>>>>>> origin/upstream-releases
       let skipDefaultBrowserCheck = false;
       if (!AppConstants.RELEASE_BUILD) {
         promptCount =
           Services.prefs.getIntPref("browser.shell.defaultBrowserCheckCount");
-        skipDefaultBrowserCheck =
-          Services.prefs.getBoolPref("browser.shell.skipDefaultBrowserCheck");
       }
+      // CLIQZ: will skip showing dialog "Set as default" on first launch
+      skipDefaultBrowserCheck =
+        Services.prefs.getBoolPref("browser.shell.skipDefaultBrowserCheck");
```

Comment:
I'd leave in inside the original block, inverse the condition, and `#if 0` the `promptCount` variable. That would minimize the difference to Mozilla's code.